### DISCUSSION TITLE
Initial Setup

### DISCUSSION THREAD

**Discussion by t1** [GraphQL-Feder/feder#2] [Show and tell]
Hi @phillip-kruger,

I just created an initial setup, and my kids provided the logo :-)

Is the hybrid setup okay for you? Quarkus doesn't like to live in a `quarkus-pom.xml`, but it works with the `war-pom.xml`, i.e. you can build a war with `mvn -f war-pom.xml`.

**Comment by phillip-kruger** [comment]
Great thanks ! How about a core module that really just contain the code and test cases, and do not build a distribution, and then another 2 projects that build the distribution ?

**Reply by t1** [reply]
That would be okay for me. I just started as simple as possible and would have split when we really need it.

**Reply by phillip-kruger** [reply]
Ok cool.

**Reply by phillip-kruger** [reply]
Should be add some issues here as to what needs to be done ?

**Reply by t1** [reply]
Yes, sure. Maybe not the first basic steps that are all straight-forward, as it would be just extra work.

**Comment by phillip-kruger** [comment]
Love the logo !!

**Comment by t1** [comment]
Just added a simple GraphQL proxy with JAX-RS and MP REST Client.

Needs a snapshot build of [WunderBar[(https://github.com/t1/wunderbar) for the tests until I do a release.

@phillip-kruger : what do you think?

**Reply by phillip-kruger** [reply]
Nice !! I'll have a look a.s.a.p !

**Reply by t1** [reply]
The first PoC works! I can fetch data from two backends in a single request. See [`GraphQLGatewayTest#shouldGetProductNameAndPrice`](https://github.com/GraphQL-Feder/feder/blob/7134ed08dd79ec68c1dc06ce4b695ea12203dcdc/src/test/java/com/github/graphql/feder/GraphQLGatewayTest.java#L92).
